practice/flutter/flutter/logic.py

```diff
@@ -0,0 +1,167 @@
+"""flutter Logic."""
+
+from . import models
+from django.contrib.auth import authenticate, login
+from django.contrib.auth.models import User
+import datetime
+from django.test import client
+from django.contrib.auth import logout
+
+
+def create_and_save_flutt(username, body, time, user_id):
+    """Creates a new Flutt object and saves to the database.
+
+    >>> create_and_save_flutt('adkinsbass', 'Radical!', datetime.datetime(2016, 9, 2, 10, 28, 34), 0)
+    >>> models.Flutt.objects.get(author='adkinsbass', body='Radical!')
+    Flutt(author='adkinsbass', body='Radical!', timestamp=2016-09-02 10:28:34+00:00, author_id=0)
+    """
+    if not body:
```

Comment:
Minor feedback: don't rely on the false-y-ness of other values. Always explicitly check for the state you want to fail `if len(body) < 1:` doesn't require the reader to remember the false-y table.